src/content/docs/cheerpj2/index.md

```diff
@@ -29,27 +43,23 @@ If you are unsure how to start, try our [tutorial](/cheerpj2/getting-started/Tut
 
 ## What is unique about CheerpJ?
 
-1. CheerpJ can convert 100% of Java including reflection and proxy class creation, with no manual intervention on the code.
+1. CheerpJ can handle 100% of Java 8 including reflection and proxy class creation, with no manual intervention on the code.
 2. CheerpJ works directly on Java bytecode, and does not require access to the Java source code.
 3. CheerpJ comes with a full Java SE runtime, inclusive of Swing/AWT. It supports audio, printing, and any other Java SE component. The runtime supports WebAssembly for optimal performance and size.
```

Comment:
Not sure "Component" even make sense in the context of Java. "Feature"? "Functionality"